source/_components/nest.markdown

```diff
@@ -65,7 +65,15 @@ Configuration variables:
 
 Currently there is a single `nest.set_mode` service available to switch between
 "away" and "home" modes. This service requires a `home_mode` param and has an
-optional `structure` param.
+optional `structure` param. While setting "away" mode, an estimated arrival time
+can also be set with `eta`, `eta_window`, and `trip_id` parameters.
+
+- **home_mode** (*Required*): `home` or `away`
+- **structure**: Structure(s). Default apply to all structures connected with Homa Assistant.
+- **eta**: Estimated Arrival Time from now.
+- **eta_window**: Optional ETA window. Default is 1 minute.
+- **trip_id**: Optional identity for your estimation, using same `trip_id` to update previous estimation.
```

Comment:
Change line to: `**trip_id** (*Optional*): Unique ID for the trip. Using an existing trip ID will update that trip's ETA.`